### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by frankdarnese**
I'm suggesting a faster and shorter way to create & cd into a new dir:

Instead of: 
```TXT
mkdir your-wordpress-folder && cd your-wordpress-folder`
```

Could we use?
```TXT
take your-wordpress-folder
```

### CODE REVIEW

**Comment by harryfinn** [comment]
@frankdarnese the `take` command is built into `oh-my-zsh` so probably best to leave this as the slightly longer command, only as to ensure that all users, regardless of whether they use `oh-my-zsh` or not, can follow this guide

File: README.md [hunk 1]
```diff
@@ -43,6 +43,10 @@ present you with the current PHP & WP-CLI versions installed if successful.
 mkdir your-wordpress-folder && cd your-wordpress-folder
 wp core download --locale=en_GB
 ```
+or if you have 'Oh my Zsh' installed you can simply use a shortcut:
```
Comment (harryfinn) [inline README.md]: Can we add a line space above and below this text, just to keep it separate from the triple back ticks? It shouldn't cause any layout issues. 

Also perhaps we can rephrase this a little to explain what `take` does : 

Or if you have 'Oh My Zsh' installed, you can simplify this with the `take` command which will create and swap into the directory:

Comment (frankdarnese) [inline README.md]: Correct! :-)

**Comment by frankdarnese** [comment]
Cool!

**Pull request merged**